cost/aws/schedule_instance/aws_schedule_instance.pt

```diff
@@ -84,6 +84,7 @@ datasource "ds_get_all_instances" do
     query "Filter.1.Name", "tag-key"
     query "Filter.1.Value.1", "schedule"
     query "Version", "2016-11-15"
+    ignore_status [403]
```

Comment:
We should removed this because it will result in no instances and no indication of an error to inform the user something is wrong.

Reply:
made the change